infrastructure-playbooks/rgw-loadbalancer.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: gather facts on rgw hosts
+  hosts:
+    - "{{ rgw_group_name|default('rgws') }}"
+  become: true
+  tasks:
+    - import_role:
+        name: ceph-defaults
+    - import_role:
+        name: ceph-facts
+    - debug: msg="gather facts on rgw hosts for following reference"
+
+- hosts: haproxys
+  vars:
+    - rgw_group_name: rgws
```

Comment:
why hardcoding the rgw_group_name here when you can reuse `"{{ rgw_group_name|default('rgws') }}"` like before ?

Reply:
@dsavineau yes, you are right.